drools-game-engine-horserace-model/src/main/java/org/drools/game/horserace/model/Team.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.drools.game.horserace.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Samuel
+ */
+public class Team {
+
+    private String name;
+    private int points;
+    private List<String> playersInTeam;
```

Comment:
We are not using this field in this game, so this is dead code. We need to make sure that we don't leave any of this